frontend/src/metabase/lib/engine.js

```diff
@@ -79,7 +79,7 @@ export function formatJsonQuery(query, engine) {
   if (engine === "googleanalytics") {
     return formatGAQuery(query);
   } else {
-    return JSON.stringify(query);
+    return JSON.stringify(query, null, 2);
```

Comment:
Note to reviewers.

I want to achieve something like this

#### When SQL is longer than the modal
![image](https://user-images.githubusercontent.com/1937582/162136458-2f5dffc6-cd7c-41f9-b216-38a38c517e91.png)

#### When SQL is shorter than the modal
![image](https://user-images.githubusercontent.com/1937582/162136551-1002a1d7-5205-4e63-a1ed-43a1b1dd1a0d.png)

I'm not sure how to do that only via CSS without resorting to JS hack.